cursorless.nvim/README.md

```diff
@@ -89,3 +117,9 @@ require('lazy').setup({
   'vim-scripts/BufOnly.vim'
 })
 ```
+
+### Sending pull requests
```

Comment:
This will need to be moved to `docs/contributing/cursorless-in-neovim.md`, along the BufOnly.vim configuration above.

Reply:
Btw, we should probably just link to that `docs/contributing/cursorless-in-neovim.md` in "Contributors" and have nothing else in that section